sharepoint/sharepoint-server-ps/sharepoint-server/New-SPWebApplication.md

```diff
@@ -38,10 +40,10 @@ For permissions and the most current information about Windows PowerShell for Sh
 
 ### ------------------EXAMPLE 1-----------------------
 ```powershell
-New-SPWebApplication -Name "Contoso Internet Site" -Port 80 -HostHeader sharepoint.contoso.com -URL "http://www.contoso.com" -ApplicationPool "ContosoAppPool" -ApplicationPoolAccount (Get-SPManagedAccount "DOMAIN\wa")
+New-SPWebApplication -Name "Contoso Internet Site" -Port 80 -HostHeader sharepoint.contoso.com -URL "https://www.contoso.com" -ApplicationPool "ContosoAppPool" -ApplicationPoolAccount (Get-SPManagedAccount "DOMAIN\jdoe")
```

Comment:
Being that the port specified is 80, is this still https in the URL param?

Reply:
"DOMAIN\wa" would be preferred rather than 'jdoe' as that sounds like an end user name vs. a service account.

Reply:
FWIW, I think "wa" is too short to be clear as a service account name.  Should we use "serviceaccount" instead?

Reply:
@TroyStarr I think we may use it in other 'best practice' documentation for SP Svr but would need to look it up. We signify 'wa' for 'WebApplication' and 'sa' for 'Service Application'.

Reply:
Unless extreme compactness is needed, I recommend a longer example account name to give a better hint of what the purpose of the account is. :-)